### PR TITLE
DIV-8427 | Fix Address Field issue in certificate

### PR DESCRIPTION
**Pull request by varadeth**

### CODE REVIEW

File: mobile/src/components/RegisterBeneficiaryForm/index.js [hunk 1]
```diff
@@ -26,15 +26,16 @@ import doseCompletedImg from "../../assets/img/dose-completed.svg";
...
     "Female",
     "Other"
 ];
-export const RegisterBeneficiaryForm = ({verifyDetails, state, onBack, onContinue, buttonText}) => {
-    const [formData, setFormData] = useState({...state});
-
+export const RegisterBeneficiaryForm = ({verifyDetails, onBack, onContinue, buttonText}) => {
+    const {state} = useWalkInEnrollment();
+    const [formData, setFormData] = useState(state);
+    debugger
```
Comment (egov-joy) [inline mobile/src/components/RegisterBeneficiaryForm/index.js:38]: remove debugger

File: mobile/src/components/RegisterBeneficiaryForm/index.js [hunk 2]
```diff
@@ -321,8 +321,13 @@ const BeneficiaryDetails = ({verifyDetails, formData, setValue, errors}) => {
                         hidden={verifyDetails}>
                     <option disabled selected={!formData.district} value>Select District</option>
                     {
-                        districts.map(d => <option selected={d.name === formData.district}
-                                                   value={d.name}>{d.name}</option>)
+                        districts.map(d => { 
+                            if(d.name.indexOf(formData.district) >= 0) {
+                                formData.district = d.name;
```
Comment (egov-joy) [inline mobile/src/components/RegisterBeneficiaryForm/index.js:326]: formData should be updated using setFormData

Comment (varadeth) [reply mobile/src/components/RegisterBeneficiaryForm/index.js]: done

File: mobile/src/components/RegisterBeneficiaryForm/index.js (hunk 1 above)
Comment (varadeth) [reply mobile/src/components/RegisterBeneficiaryForm/index.js]: removed

**Pull request merged**